packages/licensing/src/class-manager.php

```diff
@@ -0,0 +1,195 @@
+<?php
+/**
+ * A Terms of Service class for Jetpack.
+ *
+ * @package automattic/jetpack-licensing
+ */
+
+namespace Automattic\Jetpack\Licensing;
+
+use Automattic\Jetpack\Connection\Manager as Connection_Manager;
+use Jetpack_IXR_ClientMulticall;
+use Jetpack_Options;
+use WP_Error;
+
+/**
+ * Class Manager.
+ *
+ * @since ??
+ *
+ * Helper class that is responsible for validating licenses to the current site.
+ */
+class Manager {
```

Comment:
In general, we've tried to move away from the `Manager` pattern in our packages, with the exception of the Connection manager. See #12645, #12644, or #12647 for some examples.

I think you could get rid of this here as well.